### DISCUSSION TITLE
Welcome to `supabase-csharp` Discussions!

### DISCUSSION THREAD

**Discussion by acupofjose** [supabase-community/supabase-csharp#37] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NiklasTaube** [comment]
Hello, I'm trying to create a SupabaseModel for my profiles table so that i can use Client.Instance.From<>(). However, the id from profiles is a foreign key referencing the id from auth.users so I am unsure how to do it. How should i do it?

**Reply by rhuanbarros** [reply]
Hi, if I get what you need right, you can create the model field in the c# like this:

`
[Column("user_id")]
public string? User_id { get; set; }
`

Later. when inserting row in the table you just need to pass the id of the user.


`
TableModel item = new()
{
User_id = "id"
}
Postgrest.Responses.ModeledResponse<TableModel> modeledResponse = await client.From<TableModel>().Insert(item);
return modeledResponse.Models;
`

**Reply by rhuanbarros** [reply]
I just created a PR with some code that could help you.

https://github.com/supabase-community/supabase-csharp/pull/40

**Reply by NiklasTaube** [reply]
Thanks, it works now.
